<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGPSlog</class>
 <widget class="QWidget" name="CGPSlog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>581</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>140</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>581</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Current position</string>
   </property>
   <widget class="QLabel" name="label_lat_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lat_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lat_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lat_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Altitude</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_lat">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_lon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_alt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
